src/components/tags/TagSelector.tsx

```diff
@@ -0,0 +1,5 @@
+function TagSelector() {
```

Comment:
What's this file for?

Reply:
It's for selecting tags--implemented in a later commit.